docs/user/connected-accounts.rst

```diff
@@ -1,28 +1,51 @@
-Connecting Your VCS Account
-===========================
+Connecting your Git repository
+==============================
+
+In this article, we explain how connecting your Read the Docs account to one of the supported Git providers
+automatically configures your Git repository and your Read the Docs project.
+
+✅️ Signed up with your Git provider?
+  If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab
+  credentials, you're all done. Your account is connected and the integration explained
+  below is enabled.
+⬇️ Signed up with your email address?
+  If you have signed up to Read the Docs with your email address,
+  you can add the connection to the Git provider afterwards.
+  Read more about how it works below.
```

Comment:
We should link at least one of these `below`'s.

Reply:
This has been fixed by adding a direct link to the how-to in this step. I think that it makes sense - either it's the prerequisite for the explanation to be relevant or it's the "action item" of understanding the explanation and wanting to switch on the integration.